src/sysinfo.cc

```diff
@@ -20,6 +20,15 @@
 #include <versionhelpers.h>
 #include <windows.h>
 
+// WINAPI_FAMILY_PARTITION is defined in winapifamily.h
```

Comment:
why did you move this?  it is only currently used here, but we try to keep all the platform definitions in one place so they're easy to find if we need to change something.

Reply:
I checked it again on win10 and found that the macro ```BENCHMARK_OS_WINDOWS_WIN32``` is not defined. This is because the original code used the macro ```WINAPI_FAMILY_PARTITION``` before it was defined in ```winapifamily.h```. The alternative is to move it back and include ```winapifamily.h``` before the use.